nss-cohort-36/bangazon-workforce-mgt-lords-of-might-magic
# Cf fix departments

# Description

[Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.]

Closes #15 

## Type of change

Please delete options that are not relevant.


# Testing Instructions

Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout <branch-name>`
4. please check that departments each display the correct size (number of employees)

# Checklist:
